packages/@storefront/flux-capacitor/src/core/emitter.ts

```diff
@@ -0,0 +1,90 @@
+import { EventEmitter } from 'eventemitter3';
+
+class Emitter extends EventEmitter {
+  _barriers: Barriers;
+  _lookups: Lookups;
+
+  constructor() {
+    super();
+    this._barriers = {};
+    this._lookups = {};
```

Comment:
TypeScript allows you to put properties right on the class:

```
class Emitter extends EventEmitter {
  _barriers: Barriers = {};
  _lookups: Lookups = {};

  // ...
}
```

Reply:
Will apply.